index.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap"
+    rel="stylesheet" />
+  <title>KMarketer</title>
+</head>
+<body>
+  <div class="header">
+      <h2 class="header__logo">KMarketer</h2>
+      <div class="header__nav-left">
+        <div class="nav__items">
+          <a class="nav__item" href="">home</a>
+          <a class="nav__item" href="" alt="">product</a>
+          <a class="nav__item" href="" alt="">pricing</a>
+          <a class="nav__item" href="" alt="">contact</a>
+          <img src="./image/Vector.svg" alt="">
+          <img src="./image/btn.svg" alt="">
+        </div>
+        <img src="./image/navbar-toggler.png" alt="">
+      </div>
+  </div>
+
+  <div class="section-1">
+    <div class="section-text  section-text--big">
+    <h2 class="section-text__heading section-text__heading--big">Experts are here solveyour business problem.</h2>
+    <p class="section-text__intro section-text__intro--big">We know how large objects will act, but things on a small scale just do not act that way.</p>
+    </div>
+      <div class="section1-button">
+      <button class="button button--primary button--big-radius">Get Quote Now</button>
+      <button class="button button--secondary button--big-radius">Get Quote Now</button>
+      </div>
+  </div>
+  <div class="features">
+    <div class="section-text ">
+      <h2 class="section-text__heading">WHY CHOOSE US</h2>
+      <p class="section-text__intro">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics</p>
+    </div>
+    
+    <div class="features__card">
+      <div class="features__card-row">
+        <div class="features__card-container">
+          <div class="features__first-items">
+            <img class="features__card-logo" src="./image/Vector2.svg" alt="">
+            <h2 class="section-text section-text__heading section-text__heading--small">Digital Marketing</h2>
```

Comment:
Putting Block, Block Element and element's modifier in one class! it is not a good practice in BEM